cmd/kube-controller-manager/app/core.go

```diff
@@ -135,8 +135,8 @@ func startNodeLifecycleController(ctx ControllerContext) (http.Handler, bool, er
 	var tpAccessors []*nodeutil.TenantPartitionManager
 	var err error
 	// for backward compatibility, when "--tenant-servers" option is not specified, we fall back to the traditional kubernetes scenario.
-	if len(ctx.ComponentConfig.NodeLifecycleController.TenantServers) > 0 {
-		tpAccessors, err = nodeutil.GetTenantPartitionManagersFromServerNames(ctx.ComponentConfig.NodeLifecycleController.TenantServers, ctx.Stop)
+	if len(ctx.ComponentConfig.NodeLifecycleController.Kubeconfigs) > 0 {
```

Comment:
This name is a bit confusion as where this KubeConfig is pointing to. Perhaps rename to TenantPartitionKubeConfig?

Reply:
fixed